src/pks/transport/transport_ats_pk.cc

```diff
@@ -513,7 +519,8 @@ Transport_ATS::SetupPhysicalEvaluators_()
   S_->Require<CompositeVector, CompositeVectorSpace>(key_, tag_next_, passwd_)
     .SetMesh(mesh_)
     ->SetGhosted(true)
-    ->AddComponent("cell", AmanziMesh::Entity_kind::CELL, num_components_);
+    ->AddComponent("cell", AmanziMesh::Entity_kind::CELL, num_components_)
+    ->AddComponent("face", AmanziMesh::Entity_kind::FACE, num_components_);
```

Comment:
See below -- I think once you use diff_sol as the variable in UpdateFluxes, you can revert this change.